src/media.rs

```diff
@@ -63,6 +63,7 @@ const TABLE: &[(&str, Media, &[&str])] = &[
   ("text/html;charset=utf-8", Media::Iframe, &["html"]),
   ("text/plain;charset=utf-8", Media::Text, &["txt"]),
   ("video/webm", Media::Video, &["webm"]),
+  ("video/mp4", Media::Video, &["mp4"]),
```

Comment:
```suggestion
  ("video/mp4", Media::Video, &[]),
```

How about this? Server can preview inscriptions with `video/mp4` content types, but can't publish them until we can do a better job of making sure they'll be viewable.

Reply:
That could work for now!